e2e/deployers/crud.go

```diff
@@ -193,9 +193,11 @@ func CreateSubscription(ctx types.Context, s Subscription) error {
 			return err
 		}
 
-		log.Info("Subscription already Exists")
+		log.Debugf("Subscription \"%s/%s\" already exists", managementNamespace, name)
```

Comment:
Here we need both to unify the "exists" and add new parameters. The exists part can be done in a first cleanup commit, and the next commit can add the new arguments, but it may not worth the trouble, so you can keep both changes in this commit.